stable_baselines3/common/type_aliases.py

```diff
@@ -14,7 +14,8 @@
 TensorDict = Dict[str, th.Tensor]
 OptimizerStateDict = Dict[str, Any]
 MaybeCallback = Union[None, Callable, List[callbacks.BaseCallback], callbacks.BaseCallback]
-LearningRateSchedule = Callable[[float], float]
+Schedule = Callable[[float], float]
+LearningRateSchedule = Schedule
```

Comment:
Why distinguish these? I think we can pass a `Schedule` to a `LearningRateSchedule` and Python will not complain, it's just an alias. We could maybe use [NewType](https://docs.python.org/3/library/typing.html#newtype) to actually force the typer to distinguish them (but that'd require the user explicitly annotating their callables as Schedule vs LearningRateSchedule).

Reply:
No good reason, `LearningRateSchedule` was introduced in https://github.com/DLR-RM/stable-baselines3/pull/215 by @erniejunior 

there was a remark here: https://github.com/DLR-RM/stable-baselines3/pull/215#discussion_r516523591
but I realized only afterward that PPO clip range was already a schedule

Reply:
any comments on that @erniejunior ?